### PR TITLE
Row- and col-span, cell background colors and hyperlink support

### LINKED ISSUE #1: Excel row- & col-span
I just stumbled upon this project and I have to say I think it is great and quite helpful!
I will reference it from docToolchain asap.

I've also seen that you noted down as open issues the implementation of row- and col-span for excel.
Have you checked out the code in docToolchain for the `exportExcel`-Task?

https://github.com/docToolchain/docToolchain/blob/master/scripts/exportExcel.gradle

It is capable of handling row- and col-spans, alignments, col-width and background colors.

### PR DESCRIPTION
**Pull request by crea-doo**
I've implemented row- and col-span support and cell background color support following the implementation used at https://github.com/docToolchain/docToolchain/blob/master/scripts/exportExcel.gradle

Additionally support for hyperlinks was added as well as dependency/plugin updates...

### CODE REVIEW

**Comment by uniqueck** [comment]
Hi @crea-doo, 

thx for this PR. It looks good to me and is some stuff we need in doctoolchain for the next generation.
Is it ok for you to provide one or more additional sample excel files with cell and row span and a hyperlink and the matching expected AsciiDoc file?

**Comment by uniqueck** [comment]
Fixes #5

File: src/test/java/org/uniqueck/asciidoctorj/exceltableconverter/ExcelTableGeneratorBlockMacroProcessorTest.java [hunk 1]
```diff
@@ -40,12 +40,11 @@ void testExcelFileExistAndSheetNameIsProvided() {
     @ParameterizedTest
     @ValueSource(strings = {"simpleExcelTable.xls", "simpleExcelTable.xlsx"})
     void testExcelFileExistAndSheetNameIsProvidedAndSheetExist(String excelFileName) {
-        String actual = convert("excel::" + excelFileName + "[sheetName=Tabelle1]");
+        String actual = convert("excel::" + excelFileName + "[sheetName=Tabelle1,options=\"header\"]");
```
Comment (uniqueck) [inline src/test/java/org/uniqueck/asciidoctorj/exceltableconverter/ExcelTableGeneratorBlockMacroProcessorTest.java:43]: If options = header is optional, you can add a test case without options = header

File: src/test/java/org/uniqueck/asciidoctorj/exceltableconverter/ExcelTableGeneratorBlockMacroProcessorTest.java [hunk 2]
```diff
@@ -40,12 +40,11 @@ void testExcelFileExistAndSheetNameIsProvided() {
     @ParameterizedTest
     @ValueSource(strings = {"simpleExcelTable.xls", "simpleExcelTable.xlsx"})
```
Comment (uniqueck) [inline src/test/java/org/uniqueck/asciidoctorj/exceltableconverter/ExcelTableGeneratorBlockMacroProcessorTest.java:41]: add more sample excel files or modify the existing one with all kind of formats like cell and row span and a cell with a hyperlink and so one.

**Review by uniqueck — CHANGES_REQUESTED** [review]

**Comment by crea-doo** [comment]
Hi! No problem - added the unit tests...

**Review by uniqueck — APPROVED** [review]

**Comment by uniqueck** [comment]
Thx @crea-doo.

**Pull request merged**